platform-sdk/platform-apps/tests/AddressBookTestingTool/src/main/java/com/swirlds/demo/addressbook/AddressBookTestingToolConfig.java

```diff
@@ -18,20 +18,24 @@
 
 import com.swirlds.config.api.ConfigData;
 import com.swirlds.config.api.ConfigProperty;
+import java.time.Duration;
 
 /**
  * Config for address books and utilities that deal with address books.
  *
- * @param softwareVersion
- * 		The integer value of the software version of the AddressBookTestingToolMain SwirldMain application.
- * @param weightingBehavior
- *      The integer value of the weighting behavior of the AddressBookTestingToolState SwirldState.
- * @param testScenario
- *     The string value of the test scenario being run for validation. This must match an enumerated value in {@link AddressBookTestScenario}.
+ * @param softwareVersion    The integer value of the software version of the AddressBookTestingToolMain SwirldMain
+ *                           application.
+ * @param weightingBehavior  The integer value of the weighting behavior of the AddressBookTestingToolState
+ *                           SwirldState.
+ * @param testScenario       The string value of the test scenario being run for validation. This must match an
+ *                           enumerated value in {@link AddressBookTestScenario}.
+ * @param freezeAfterGenesis if not 0, describes a moment in time, relative to genesis, when a freeze is scheduled
+ *                           {@link com.swirlds.common.config.StateConfig#savedStateDirectory()}.
```

Comment:
Do you mean to link `savedStateDirectory()` here? I don't see how it applies

Reply:
I copied the comment from what was present in the `ConsistencyTestingToolConfig`.   Should the link there be deleted as well?   I don't see how it applies either. :-)

Reply:
Lol, so all of the things I found on this PR were just copy-pasted from my own code :D